fairscale/utils/testing.py

```diff
@@ -209,8 +210,20 @@ def worker_process(
 
     initialize_model_parallel(1, world_size, **kwargs)
 
+    # Make sure that CUDA operations are repeatable
+    context = (
+        torch.backends.cudnn.flags(benchmark=False, deterministic=True)  # type: ignore
+        if torch.cuda.is_available() and hasattr(torch.backends.cudnn, "flags")
+        else contextlib.suppress()
+    )
+
+    if torch.cuda.is_available() and not hasattr(torch.backends.cudnn, "flags"):
+        torch.backends.cudnn.benchmark = False
+        torch.backends.cudnn.deterministic = True
+
     try:
-        func(*args)
+        with context:
```

Comment:
Since this seems to be a generic testing function used in other places in fairscale, are you sure there is no performance concern with being in deterministic mode?

Reply:
It's a good catch, I don't think that we use this for perf measurement but it was a little intrusive.. fixing that